godot-core/src/registry/property.rs

```diff
@@ -549,3 +554,31 @@ pub(crate) fn builtin_type_string<T: GodotType>() -> String {
         format!("{}:{}", variant_type.sys(), T::godot_type_name())
     }
 }
+
+/// Creates `hint_string` to be used for given `GodotClass` when used as an `ArrayElement`.
+pub(crate) fn object_export_element_type_string<T, S>(class_hint: S) -> String
+where
+    T: GodotClass,
+    S: Display,
+{
```

Comment:
If you use `impl Display` instead of a named generic parameter `S`, the call site won't have to specify the type of the 2nd generic argument.